examples/base_difference_mapcubes.py

```diff
@@ -0,0 +1,41 @@
+"""
+========================================
+Base Difference in Mapcubes
+========================================
+
+"""
+############################################################################
+
+# Importing Libraries
+import numpy as np
+import matplotlib.pyplot as plt
+from datetime import date
+import matplotlib.colors as colors
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+# Forming the maps
+aia1 = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE)
+aia2 = sunpy.map.Map(AIA_193_CUTOUT02_IMAGE)
+aia3 = sunpy.map.Map(AIA_193_CUTOUT03_IMAGE)
+
+# base map
+base = aia1
+maps = sunpy.map.MapCube(aia1,aia2,aia3)
+diffmap = []
+for map_i in maps:
+    aiamap1_rot = diffrot.diffrot_map(base, time=map_i.date)
```

Comment:
The diffrot is the wrong way round.  Maps at different times should be differentially rotated to the time of the base map.  This ensures that the coordinates in each differentially rotated map are the same.